modules/apps/journal/journal-web/src/main/resources/META-INF/resources/article/basic_info.jsp

```diff
@@ -101,11 +101,20 @@ DDMStructure ddmStructure = journalEditArticleDisplayContext.getDDMStructure();
 		</div>
 
 		<aui:script>
-			Liferay.Util.disableToggleBoxes(
-				'<portlet:namespace />autoArticleId',
-				'<portlet:namespace />newArticleId',
-				true
+			var autoArticleInput = document.getElementById(
+				'<portlet:namespace />autoArticleId'
 			);
+			var newArticleInput = document.getElementById(
+				'<portlet:namespace />newArticleId'
+			);
+
+			if (autoArticleInput && newArticleInput) {
+				newArticleInput.disabled = true && autoArticleInput.checked;
```

Comment:
Hmm, this line is a bit odd. Why is there an explicit `true` value here?

Reply:
Well, sure, now that I look at it after not staring at it for hours it doesn't make sense. But I was basically trying to replicate the old implementation. But yea, looks dumb now that I see it again haha.